src/Features/CSharp/Portable/Snippets/AbstractCSharpAutoPropertySnippetProvider.cs

```diff
@@ -64,7 +65,7 @@ protected override async Task<PropertyDeclarationSyntax> GenerateSnippetSyntaxAs
             type: compilation.GetSpecialType(SpecialType.System_Int32).GenerateTypeSyntax(allowVar: false),
             explicitInterfaceSpecifier: null,
             identifier: identifierName.ToIdentifierToken(),
-            accessorList: SyntaxFactory.AccessorList([.. accessors.Where(a => a is not null)!]));
+            accessorList: SyntaxFactory.AccessorList([.. (IEnumerable<AccessorDeclarationSyntax>)accessors.Where(a => a is not null)]));
```

Comment:
why was this needed?

Reply:
Without the explicit cast, `accessors.Where(a => a is not null)` is considered `IEnumerable<AccessorDeclarationSyntax?>`, and `!` does not suppress the warning from `..`.

Reply:
It feels like `!` placed on the expression where the warning is reported should suppress that warning. If it is not doing that, we should file a follow-up issue.

Reply:
https://github.com/dotnet/roslyn/issues/75560